labelplus/gtkui/common/gtklib/dnd.py

```diff
@@ -242,9 +242,7 @@ def remove_target(self, name):
 
   def _do_drag_button_press(self, widget, event):
 
-    _, x, y, _ = widget.get_bin_window().get_device_position(event.get_source_device())
-    pos = widget.convert_bin_window_to_tree_coords(x, y)
-    if pos[1] < 0:
+    if event.y < 0:
```

Comment:
I don't really understand what these lines were doing and if they are even needed. I just realized that the segfault was coming from the `get_device_position` method.
`pos` variable is not needed anymore so I thought about just removing the 4 lines, but `event.y` seems similar to `pos[1]`, so it can be a good replacement.